### ISSUE TITLE
Metadata: Support a separate digitization date (for scans of old and new film photographs alike)

### ISSUE THREAD

**Issue by GlassedSilver** [photoprism/photoprism#3306]
@GlassedSilver PhotoPrism does not currently support a separate digitization date. We focus mainly on digital photography, where CreateDate and DateTimeOriginal should be approximately the same. 

We know this is not the case for e.g. scanned images. You are welcome to open a separate feature request for this.

_Originally posted by @graciousgrey in https://github.com/photoprism/photoprism/issues/3220#issuecomment-1485218378_
           
Probably depends on #3220 as well.

**Comment by lastzero** [comment]
- Since you can digitize the same picture several times, e.g. in different quality, I assume that the digitization date should be an optional field for individual files? That will be relatively easy to add.
- It would be more difficult to sort by that field if you needed to. That's because the current queries and database index are optimized for the creation date (when the pictures were originally taken). This is the same for related files in a stack, even if there are multiple versions, some of which may have been digitized or edited at a later date.

**Comment by GlassedSilver** [comment]
1)
Yeah I don't know the needs and workflows of others, but the digitization date is mostly a thing you refer to on demand, sorting by it, at least as far as being able to sort by a combination of digitized + creation as fallback isn't really needed - at least for me.

2)
If anything one might consider such a thing as a long-term stretch goal if there is a need for it.


As far as stacks are concerned, yeah they are an interesting case indeed, especially when you have various versions of the same picture that ultimately make for very different images (as opposed to photographs).

A different scan/development (film or digital RAW) yields different **images**. But I digress and if I'm not mistaken I have brought up this issue before (or I dreamt having done so :P )
